benjaminbwright/project-2-starter
# 3 setup prettier eslint integration for code quality

## Purpose

Please provide a detailed explanation of the PR. What existing problem does it solve?

Fixes #3 

## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

- .eslintignore
- .eslintrc.json
- .prettierrc.json
- .prittierignore

## Steps to Reproduce or Test

Checkout the branch for this pr and add double quotes to a string in a file. You should get output in the problems console and redlines from eslint vscode extension.
